src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,69 @@
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
```

Comment:
What are the changes in this file for?

Reply:
This file adds the security constraint so that it will redirect the user that is not logged in to log in if they try to access personal pages. It also specifies the servlets/url-patterns which the SessionFilter should be applied to.